### PR TITLE
Nbart aws change

### PR DESCRIPTION
**Pull request by kieranricardo**

### CODE REVIEW

File: scripts/upload_s2_nbart.py [hunk 1]
```diff
@@ -110,7 +111,7 @@ def upload_granule(granule_id):
 
         publish_sns(SNS_ARN, stac_dump, message_attributes, session=session)
 
-        s3_dump(stac_dump, s3_stac_path)  # upload STAC last
+        s3_dump(stac_dump, s3_stac_path, ACL="bucket-owner-full-control")  # upload STAC last
```
Comment (alexgleith) [inline scripts/upload_s2_nbart.py:114]: Can you please set ContentType for this.
```suggestion
        s3_dump(stac_dump, s3_stac_path, ACL="bucket-owner-full-control", ContentType="application/json")  # upload STAC last
```

File: scripts/upload_s2_nbart.py [hunk 2]
```diff
@@ -150,7 +151,11 @@ def upload_metadata(granule_id):
     )
     stac_dump = json.dumps(stac, default=json_fallback, indent=4)
 
-    s3_dump(yaml.safe_dump(serialise.to_doc(eo3), default_flow_style=False), s3_eo3_path)
+    s3_dump(
+        yaml.safe_dump(serialise.to_doc(eo3), default_flow_style=False), 
+        s3_eo3_path, 
+        ACL="bucket-owner-full-control",
+    )
```
Comment (alexgleith) [inline scripts/upload_s2_nbart.py:158]:
```suggestion
    s3_dump(
        yaml.safe_dump(serialise.to_doc(eo3), default_flow_style=False), 
        s3_eo3_path, 
        ACL="bucket-owner-full-control",
        ContentType="text/vnd.yaml"
    )
```

**Review by alexgleith — CHANGES_REQUESTED** [review]
Two changes around content type, which is really nice to have on S3.

It would be awesome to upgrade this to use the same functions as you're using here https://github.com/GeoscienceAustralia/dea-airflow/blob/master/scripts/c3_to_s3_rolling.py

**Comment by kieranricardo** [comment]
@alexgleith yeah I'd like to clean that script up but I'd prefer to leave to another PR - I don't wanna make any potentially breaking changes here!

**Review by alexgleith — APPROVED** [review]

**Comment by NikitaGandhi** [comment]
@kieranricardo what about this upload step? 

```
    execute_upload = SSHOperator(
        task_id='execute_upload',
        # language="Shell Script"
        command=dedent(COMMON + """
            {% set aws_creds = params.aws_hook.get_credentials() -%}
            # Export AWS Access key/secret from Airflow connection module
            export AWS_ACCESS_KEY_ID={{aws_creds.access_key}}
            export AWS_SECRET_ACCESS_KEY={{aws_creds.secret_key}}
            python3 '{{ work_dir }}/upload_s2_nbart.py' granule_ids.txt
        """),
        remote_host='gadi-dm.nci.org.au',
        params={'aws_hook': aws_hook},
        timeout=10 * HOURS,
    )
    [upload_uploader_script, generate_list, upload_utils] >> execute_upload
```

**Comment by kieranricardo** [comment]
@NikitaGandhi are you referring to exporting the aws creds and them ending up in the airflow logs?

**Comment by NikitaGandhi** [comment]
> @NikitaGandhi are you referring to exporting the aws creds and them ending up in the airflow logs?



> @NikitaGandhi are you referring to exporting the aws creds and them ending up in the airflow logs?

can we create an issue ticket for this?

**Comment by kieranricardo** [comment]
https://gajira.atlassian.net/browse/DEADM-1604?atlOrigin=eyJpIjoiOGUzOWI1N2VhNjk4NDMzOGE5Zjk2ZTgyZjI3YTIzNjIiLCJwIjoiaiJ9

are you okay for me to merge this @NikitaGandhi?

**Review by NikitaGandhi — APPROVED** [review]

**Pull request merged**